test/fixtures/cookbooks/mingw_tdm_install/recipes/install64.rb

```diff
@@ -16,9 +16,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-include_recipe 'seven_zip::default'
+include_recipe 'mingw::default'
 
-root_path = node['mingw']['path64']
+root_path = "#{ENV['SYSTEMDRIVE']}\\mingw64"
```

Comment:
Agreed - but I still need to explicitly set the root of mingw_get for now - so it doesn't save me that much.  Once we're happy with some layout and I'm sure that we won't need to screw with things, then I'll go add some defaults.